polaris-for-vscode/scripts/prepublish.js

```diff
@@ -1,10 +1,9 @@
 const fs = require('fs');
 const path = require('path');
 
-const {
-  getCustomPropertyNames,
-  getGroupedCustomPropertyCompletionItems,
-} = require('../../polaris-react/scripts/utilities/getCustomPropertyNames');
+const getCustomPropertyNames = require('../../polaris-tokens/scripts/getCustomPropertyNames');
+
+const getGroupedCustomPropertyCompletionItems = require('./getGroupedCustomPropertyCompletionItems');
```

Comment:
This `prepublish` script should be completely removed in favor of adding `@shopify/polaris-tokens` as a dependency and consuming it's modules and assets directly.